lib/src/functions_client.dart

```diff
@@ -13,6 +13,26 @@ class FunctionsClient {
   final YAJsonIsolate _isolate;
   final bool _hasCustomIsolate;
 
+  /// Getter for the headers
+  Map<String, String> get headers {
+    return _headers;
+  }
+
+  /// Sets the headers to be sent with requests.
+  /// If an 'Authorization' header is already present,
+  /// it will be kept unless removeAuth is called first.
+  /// @param value a Map containing the new headers to set.
```

Comment:
I think this might be the preferred format in Dart!
```suggestion
  /// [value] a Map containing the new headers to set.
```